assignments/arrays.js

```diff
@@ -63,32 +63,46 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log('The last car is a ' + inventory[32].car_year + ' ' + inventory[32].car_model + ' ' + inventory[32].car_make);
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory[(inventory.length)-1];
+console.log('Car 33 is a ' + lastCar.car_year + ' ' + lastCar.car_model + ' ' + lastCar.car_make);
 
```

Comment:
template literals can help make this code easier to write here is an example. I would like to challenge to go back and rewrite using template literals (only if you have time and want the challenge). 
console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}` );